recipes/r-nlsic/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-nlsic
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/nlsic_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/nlsic/nlsic_{{ version }}.tar.gz
+  sha256: e8601bb99230966b6bbf827193a4b4005b9a6150a5b5c58f4e830d57bff9c9bf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-nnls
+  run:
+    - r-base
+    - r-nnls
+    - r-numderiv
```

Comment:
Where does that come from?
```suggestion
```

Reply:
r-base and r-nnls were generated by skeleton (NNLS is a package imported to nlsic so its presence is indeed mandatory).
I have added r-numderiv as this package is in Suggestions of nlsic.

Reply:
please remove: while it is possible to add suggested packages as run dependency, the conda-forge cran packages generally don't do that. So the user can decide, if he wants the extra package installed or not.

Reply:
done